uxarray/plot/dataarray_plot.py

```diff
@@ -61,3 +71,159 @@ def datashade(uxda: UxDataArray,
         _cmap = cmap
 
     return tf.shade(aggregated, cmap=_cmap, **kwargs)
+
+
+def _point_raster(uxda: UxDataArray,
```

Comment:
positional args, i.e.  `*args` here

Reply:
I'm still a bit hesitant to why we need to include `*args` in our function definition, as I don't see a use-case where having a non-fixed amount of positional arguments would ever be used in any of the functions that we call, unless it's a direct wrapper to a plotting method.

I've removed it from the rasterize function unless we can come up with a valid use case for it.

Reply:
Ok, that makes sense, we might likely never need positional args.